Prox/Prox/AppDelegate.swift

```diff
@@ -163,7 +163,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             let placeKey = notification.userInfo?[notificationEventPlaceIDKey] as? String {
             if ( application.applicationState == .inactive || application.applicationState == .background  ) {
                 placeCarouselViewController?.openPlace(placeKey: placeKey, forEventWithKey: eventKey)
-                Analytics.logEvent(event: AnalyticsEvent.EVENT_NOTIFICATION, params: [AnalyticsEvent.PARAM_ACTION: AnalyticsEvent.BACKGROUND])
+                Analytics.logEvent(event: AnalyticsEvent.EVENT_CLICKED, params: [AnalyticsEvent.PARAM_ACTION: AnalyticsEvent.BACKGROUND])
```

Comment:
Talking with @fluffyemily atm but [this SO post](http://stackoverflow.com/q/5099483) seems to indicate there are two entry points to the user from a user pressing a notification:

1. App is not running (via `didFinishLaunchingWithOptions`)
1. App is running in the BG (via `didReceive`)

It's possible `didReceive` is called in both cases.

fwiw, we don't have code for the former case to open the specific place so we should probably land this and add analytics when we fix it.